custom_model_runner/requirements.txt

```diff
@@ -26,3 +26,4 @@ termcolor
 packaging
 markupsafe<=2.1.3
 pydantic==2.9.2
+datarobot-storage
```

Comment:
I think this is only available internally, are we sure we can add this dependency?

Reply:
Would be good to pin to specific version too

Reply:
Yes, we can use it - there was a decision to make it public.

Reply:
Ok, I'll allow any versions up to a major upgrade.